tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -2877,6 +2877,14 @@ TEST_F(CompilerTest, CodeGenWaveSize) {
   CodeGenTestCheck(L"attributes_wavesize.hlsl");
 }
 
+TEST_F(CompilerTest, CodeGenVectorIsnan) {
+  CodeGenTestCheck(L"isnan_vector_argument.hlsl");
+}
+
+TEST_F(CompilerTest, CodeGenVectorAtan2) {
+  CodeGenTestCheck(L"atan2_vector_argument.hlsl");
+}
+
```

Comment:
The TAEF version of this file requires TEST_METHOD invocations in the class above around line 158. That's why the windows builds are failing

Reply:
done